batch/batch/batch_configuration.py

```diff
@@ -12,3 +12,6 @@
 WORKER_LOGS_BUCKET_NAME = os.environ['HAIL_WORKER_LOGS_BUCKET_NAME']
 HAIL_SHA = os.environ['HAIL_SHA']
 HAIL_SHOULD_PROFILE = os.environ.get('HAIL_SHOULD_PROFILE') is not None
+ENABLE_STANDING_WORKER = bool(int(os.environ['ENABLE_STANDING_WORKER']))
```

Comment:
I prefer a consistent style for boolean environment variables. Let's do what HAIL_SHOULD_PROFILE did, the variable merely needs to be set to anything at all.